README.rst

```diff
@@ -106,14 +108,3 @@ The list of builtins that are checked for are:
 * ``sorted``
 * ``sum``
 * ``tuple``
-
-C408: Unnecessary (dict/list/tuple) call - rewrite as a literal.
```

Comment:
I presume that this removal is unintentional after the rebase? Should a section for C409-C410 be added?